src/Report/EnabledMembers.php

```diff
@@ -0,0 +1,206 @@
+                // secondary query below to get the list of method names the user has registered. So to compensate we
+                // need to bump up the entered number by 1.
+                $params['Count'] = $desiredCount + 1;
+            }
+        }
+
+        $filteredList = ArrayList::create([]);
+        $backupClass = get_class(singleton(MethodRegistry::class)->getBackupMethod());
+        /** @var Member[]&MemberExtension[] $members */
+        $members = $this->applyParams(Member::get(), $params)->toArray();
+        foreach ($members as $member) {
+            $defaultMethod = $member->getDefaultRegisteredMethod();
+            $defaultMethodClassName = $defaultMethod ? $defaultMethod->MethodClassName : '';
+
+            $registeredMethods = $member
+                ->RegisteredMFAMethods()
+                ->exclude('MethodClassName', $backupClass)
+                ->column('MethodClassName');
+            $registeredMethodNames = array_map(function (string $methodClass) use ($methods) {
+                return $methods[$methodClass];
+            }, $registeredMethods);
+
+            $memberReportData = ArrayData::create();
+            foreach ($member->summaryFields() as $field => $name) {
+                $memberReportData->$field = $member->$field;
+            }
+
+            $memberReportData->methodCount = (string)(count($registeredMethods) ?: '');
+            $memberReportData->registeredMethods = implode(', ', $registeredMethodNames);
+            $memberReportData->defaultMethodName = $methods[$defaultMethodClassName] ?? '';
+            $memberReportData->skippedRegistration = $member->HasSkippedMFARegistration
+                ? $member->dbObject('HasSkippedMFARegistration')->Nice()
+                : '';
+
+            $filteredList->push($memberReportData);
+        };
+        if (isset($countFilter)) {
+            $filteredList = $filteredList->filter('methodCount', $countFilter);
+        }
+        $this->extend('updateSourceRecords', $filteredList);
+        return $filteredList;
+    }
+
+    public function columns()
+    {
+        $columns = singleton(Member::class)->summaryFields() + [
+            'methodCount' => _t(__CLASS__ . '.COLUMN_METHOD_COUNT', 'Number of registered methods'),
+            'registeredMethods' => _t(__CLASS__ . '.COLUMN_METHODS_REGISTERED', 'Registered methods'),
+            'defaultMethodName' => _t(__CLASS__ . '.COLUMN_METHOD_DEFAULT', 'Default method'),
+            'skippedRegistration' => _t(__CLASS__ . '.COLUMN_SKIPPED_REGISTRATION', 'Skipped registration'),
+        ];
+        $this->extend('updateColumns', $columns);
+        return $columns;
+    }
+
+    /**
+     * Provides the fields used to gather input to filter the report
+     *
+     * @return FieldList
+     */
+    public function parameterFields()
+    {
+        $methods = $this->getMethodClassToTitleMapping();
+        unset($methods[get_class(singleton(MethodRegistry::class)->getBackupMethod())]);
```

Comment:
I agree with @ScopeyNZ's previous suggestion to centralise this logic. You said you didn't see the value in doing it because it's only used twice, but I see the value being that it's easier to understand what it does if you read `unset($this->getBackupMethodClassName())` instead of this line which has a little higher cognitive load, especially since you need to either assume, or delve into the code a little to work out why you're calling get_class on it